src/components/MonitorList.vue

```diff
@@ -10,7 +10,7 @@
                     <font-awesome-icon icon="times" />
                 </a>
                 <form>
-                    <input v-model="searchText" class="form-control search-input" :placeholder="$t('Search...')" autocomplete="off" />
+                    <input v-model="searchText" class="form-control search-input" :placeholder="$t('Search...')" autocomplete="off" aria-label="Search monitored sites"/>
```

Comment:
```suggestion
                    <input v-model="searchText" class="form-control search-input" :placeholder="$t('Search...')" autocomplete="off" aria-label="Search monitored sites" />
```
Could you fix the linting warning